### PR TITLE
Change comm pref consent form wording

### LINKED ISSUE #6255: OSS registration consent form
We want to change the signup process in lakeFS open source to as follows:
### Communication Preferences:
Please provide your email below:
Email*
[name@company.com           ]

[x] I'd like to receive security, product, and feature updates
[Continue]

@eladlachmi I'll also touch base with you about this in person
@iddoavn for your reference the changes we discussed

### PR DESCRIPTION
**Pull request by itaiad200**
closes #6255 

Tested manually

image

@keren-lakeFS please review to approve this meet your requirement
@ozkatz please review for product aspects

### CODE REVIEW

**Review by eladlachmi — APPROVED** [review]
LGTM

**Comment by N-o-Z** [comment]
@itaiad200 Did you consider the required changes in https://github.com/treeverse/lakeFS/issues/6204?
How would this change integrate with the required changes there?

File: webui/src/pages/setup/userConfiguration.tsx [hunk 1]
```diff
@@ -60,20 +55,13 @@ export const UserConfiguration: FC<UserConfigurationProps> = ({
...
 
-                            <Card.Text>
-                            Please provide your email address, and indicate what optional communications you&apos;d like to receive:
-                            </Card.Text>
                             <Form.Group controlId="user-email" className="mt-4">
                                 <Form.Label>Email <span className="required-field-label">*</span></Form.Label>
                                 <Form.Control type="email" placeholder="name@company.com" value={userEmail}  onChange={handleEmailChange} />
                             </Form.Group>
 
-                            <Form.Group controlId="updates-check" className="mt-4">
```
Comment (keren-lakeFS) [inline webui/src/pages/setup/userConfiguration.tsx:71]: Please change from 
`Please provide your email address, and indicate what optional communications you&apos;d like to receive:`
to 
`...`

**Review by keren-lakeFS — APPROVED** [review]

File: webui/src/pages/setup/userConfiguration.tsx (hunk 1 above)
Comment (itaiad200) [reply webui/src/pages/setup/userConfiguration.tsx:71]: Done

**Review by ozkatz — APPROVED** [review]
LGTM!

**Pull request merged**